src/utils/test-utils.tsx

```diff
@@ -6,15 +6,14 @@ import Providers from 'src/components/Providers'
 import { createPreloadedStore, store } from 'src/store'
 import { history } from 'src/routes/routes'
 import theme from 'src/theme/mui'
-import { makeProvider } from 'src/logic/wallets/store/model/provider'
 import { SafeReducerMap } from 'src/logic/safe/store/reducer/types/safe'
 import makeSafe from 'src/logic/safe/store/models/safe'
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 function renderWithProviders(Components: ReactElement, customState?: any): RenderResult {
   const customStore = {
     ...customState,
-    providers: makeProvider(customState?.providers),
+    providers: customState?.providers,
```

Comment:
If `makeProvider` is not necessary any more, wouldn't the spread above just assign `customState.providers` to `providers` already?

Reply:
Good catch! I've removed it

Reply:
Resolving after confirming so with @dasanra.